packages/lab/src/checkbox/CheckboxIcon.css

```diff
@@ -1,29 +1,26 @@
 /* Styles applied to root component */
 .salt-density-high {
-  --checkbox-size: 12px;
   --checkbox-density-bar-y: 5px;
   --checkbox-density-bar-height: 3px;
 }
 
 .salt-density-medium {
-  --checkbox-size: 14px;
   --checkbox-density-bar-y: 6px;
   --checkbox-density-bar-height: 2px;
 }
 
 .salt-density-low {
-  --checkbox-size: 16px;
   --checkbox-density-bar-y: 6px;
   --checkbox-density-bar-height: 2px;
 }
 
 .salt-density-touch {
-  --checkbox-size: 18px;
   --checkbox-density-bar-y: 6px;
   --checkbox-density-bar-height: 2px;
 }
```

Comment:
I think we might be able to get rid of all of the above. what do you think?

Reply:
I'll have to find a characteristic that gives the same values

Reply:
the density bar ones are so random, probably incorrect. the design spec should be updated with the right sizes for everything soon